src/Nervos/Address.h

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../Data.h"
+#include "../PublicKey.h"
+
+#include <string>
+
+using namespace TW;
+
+namespace TW::Nervos {
+
+enum HashType { Data0 = 0, Type1 = 1, Data1 = 2 };
```

Comment:
One-entry per line format is more readable for this enum as well

Reply:
Done